emsurfer/src/polyfills.ts

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
[yellow]
I think we're stuck with this file but let's at least remove all the comments.

Reply:
Done